### PR TITLE
Added vulkan support

### PR DESCRIPTION
**Pull request by jayrulez**

### CODE REVIEW

**Comment by nem0** [comment]
Please, add unit tests too

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 HEAD of master is not stable, for stable version use [this](https://github.com/nem0/LumixEngine/tree/f77a0b6a899be4736f0ce26133bbe8061a3828f1)
 
-3D Game Engine
+3D Game Engine with Vulkan support.
```
Comment (nem0) [inline README.md:11]: we did not use full stop here before and I don't think we should start with it now. It's too dangerous.

**Comment by nem0** [comment]
Is this dead?

**Comment by jayrulez** [comment]
Yes, dead for now. I will submit a new PR in the future with a full stop.

**Comment by nem0** [comment]
Cool, you will become one of the biggest lunex contributors if you do that.

**Comment by AnantTiwari-Naman** [comment]
Is this supposed to be passive-agressive?

**Comment by nem0** [comment]
No, no. It's as serious as the pull request.

**Pull request closed without merging**